backend/curriculum_tracking/api_views.py

```diff
@@ -902,3 +902,22 @@ class BurnDownSnapShotViewset(viewsets.ModelViewSet):
     filter_backends = [DjangoFilterBackend]
     filterset_fields = ["user__id", "timestamp"]
 
+
+
+class ReviewTrustsViewSet(viewsets.ModelViewSet):
+
+    permission_classes = [permissions.IsAdminUser
+        | ActionIs('list')
+        & (
+            core_permissions.IsCurrentUserInSpecificFilter("user")
+            | core_permissions.HasObjectPermission(
+                permissions=Team.PERMISSION_VIEW,
+                get_objects=_get_teams_from_user_filter("user"),
+            )
+        )
+    ]
+
+    queryset = models.ReviewTrust.objects.all()
```

Comment:
one small change. Can you always add an order_by on the queryset? Otherwise weird random things happen.

if there is no obvious field to order by then just do `order_by('pk')`